clients/naming_client/host_reactor_test.go

```diff
@@ -129,3 +134,90 @@ func BenchmarkHostReactor_GetServiceInfoConcurrent(b *testing.B) {
 		_, _ = client.hostReactor.GetServiceInfo(param.ServiceName, param.ClusterName)
 	}
 }
+
+func TestHostReactor_isServiceInstanceChanged(t *testing.T) {
```

Comment:
Unit test should have unordered test case.

Reply:
ok,i'll create more unit test.

Reply:
done